src/Microsoft.Data.Analysis/ReadOnlyDataFrameBuffer.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Data.Analysis
     /// </summary>
     /// <typeparam name="T"></typeparam>
     internal class ReadOnlyDataFrameBuffer<T>
-        where T : struct
+        where T : unmanaged
```

Comment:
This is a very good change. Without it, someone could pass a struct that contains reference fields, and the unsafe conversion would almost certainly cause a runtime crash, and possible data corruption.